src/patternfly/components/Drawer/drawer.scss

```diff
@@ -193,6 +197,7 @@ $pf-c-drawer__panel--list--width: (25, 33, 50, 66, 75, 100);
 
 // Modified content children
 .pf-c-drawer__body {
+  height: 100%;
```

Comment:
Adding this here disallows multiple `__body` sections where the last section stretches the remaining height https://github.com/patternfly/patternfly/pull/3220/files#diff-56492175f881007db175cb81ee985735L225. I think it should be
```
.pf-c-drawer__body:only-child {
  height: 100%;
}